boards/opentitan/README.md

```diff
@@ -0,0 +1,41 @@
+OpenTitan RISC-V Board
+=================
+
+- https://opentitan.org/
+
+OpenTitan is the first open source project building a transparent, high-quality reference design and integration guidelines for silicon root of trust (RoT) chips.
+
+Tock currently supports the OpenTitan snapshot-20191101-2 release on a Nexys Video FPGA, as described here: https://docs.opentitan.org/doc/ug/getting_started_fpga/index.html.
+
+You can get started with OpenTitan using either the Nexys Video FPGA board or simulation. See the OpenTitan [getting started](https://docs.opentitan.org/doc/ug/getting_started/index.html) for more details.
+
+Programming
+-----------
+
+Currently `make flash` is not supported and you will need to use the [OpenTitan development flow](https://docs.opentitan.org/doc/ug/getting_started_fpga/index.html#testing-the-demo-design) to flash the image.
```

Comment:
Is this still true or fixed (or really, can either this be updated, or the `flash` target be updated to print out a pointer to this explanation)?